geowebcache/s3storage/src/main/java/org/geowebcache/s3/S3BlobStoreConfig.java

```diff
@@ -266,6 +269,28 @@ public String getProxyPassword() {
     public void setProxyPassword(String proxyPassword) {
         this.proxyPassword = proxyPassword;
     }
+    
+    /**
+     * Checks access type
+     *
+     * @return public or private access
+     */
+    public CannedAccessControlList getAccessControlList() {
+        return accessControlList;
```

Comment:
You might be returning null here if it was never set.  The code that calls it never does a null check.  You should either set a default here, or mark this as potentially null using the `@Nullable` annotation or the `Optional` wrapper type and then check and handle the default situation.